Funcky/Monads/Option/Option.ListPattern.cs

```diff
@@ -0,0 +1,25 @@
+using System.Diagnostics;
+using static System.Diagnostics.DebuggerBrowsableState;
+
+namespace Funcky.Monads;
+
+public readonly partial struct Option<TItem>
+    where TItem : notnull
+{
+    private const int NoneLength = 0;
+    private const int SomeLength = 1;
+
+    [Pure]
+    [DebuggerBrowsable(Never)]
+    public int Length
```

Comment:
I *think* Count would be more applicable here?

Reply:
Fixed in: 2eb0c0fb8d264ab4a247bcac32ea1267d7bfb1df